docs/data/pages.ts

```diff
@@ -311,6 +311,7 @@ const pages: MuiPage[] = [
           { pathname: '/x/react-date-pickers/shortcuts' },
         ],
       },
+      { pathname: '/x/react-date-pickers/playground', title: 'Playground' },
```

Comment:
We can save the title, there is a default title case logic based on the pathname that we can rely on:

```suggestion
      { pathname: '/x/react-date-pickers/playground' },
```

Changed in 8757e19e01795eacd190a6027ef7bb2caccfa9bc.